documentation/modules/rn-2.6.0.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+
+This release has the following technical changes:
+
+.Technical change
+
+Technical change
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
```

Comment:
besides the `RH-SDL` items that I commented on below, the other items are high-level epics. in MTV we don't work with an epic per-feature - the features should be taken from the items within the epics. I'll add those.

Reply:
not sure what I referred to when writing MTV-587 - I'll look for a missing feature..

Reply:
> not sure what I referred to when writing MTV-587 - I'll look for a missing feature..

ok, MTV-587 was taken by mistake from MTV-728 (its title starts with "Forklift-587") so I dropped it